src/main/scala/org/apache/spark/sql/execution/datasources/greenplum/GreenplumUtils.scala

```diff
@@ -106,8 +106,13 @@ object GreenplumUtils extends Logging {
       schema: StructType,
       options: GreenplumOptions): Unit = {
     val randomString = UUID.randomUUID().toString.filterNot(_ == '-')
+    val canonicalTblName = TableNameExtractor.extract(options.table)
```

Comment:
Can we just filter the '\"' from the name of table

Reply:
If the table name is in format  likes "163"."table", we can not filter the  \" , because 163.table is illegal.

Reply:
can we then split it?

Reply:
In this Pr, I split the options.table with  schema name and raw table name.
For the schema name, if it has double quotes, they are reserved, for safeguard meantime.
For the raw table name, because we need append a random string at its end, so I must remove its double quotes.
So, I think the current process is appropriate.